docs/data-cookbooks/cnm-workflow.md

```diff
@@ -68,137 +68,171 @@ Provided the prerequisites have been fulfilled, you can begin adding the needed
+              "cumulus_message": {
+                "outputs": [
+                  {
+                    "source": "{$.granules}",
+                    "destination": "{$.meta.input_granules}"
+                  },
+                  {
+                    "source": "{$}",
+                    "destination": "{$.payload}"
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "Type": "Task",
+        "Resource": "${module.cumulus.sync_granule_task_lambda_function_arn}",
+        "Retry": [
+          {
+            "ErrorEquals": [
+              "States.ALL"
+            ],
+            "IntervalSeconds": 10,
+            "MaxAttempts": 3
+          }
+        ],
+        "Catch": [
+          {
+            "ErrorEquals": [
+              "States.ALL"
+            ],
+            "ResultPath": "$.exception",
+            "Next": "CnmResponse"
+          }
+        ],
+        "Next": "CnmResponse"
+      },
+      "CnmResponse": {
+        "Parameters": {
+          "cma": {
+            "event.$": "$",
+            "task_config": {
+              "OriginalCNM": "{$.meta.cnm}",
+              "CNMResponseStream": "ADD YOUR RESPONSE STREAM HERE",
+              "region": "us-east-1",
+              "WorkflowException": "{$.exception}",
+              "cumulus_message": {
+                "outputs": [
+                  {
+                    "source": "{$}",
+                    "destination": "{$.meta.cnmResponse}"
+                  }
+                ]
+              }
+            }
+          }
+        },
+        "Type": "Task",
+        "Resource": "${aws_lambda_function.cnm_response_task.arn}",
+        "Retry": [
+          {
+            "ErrorEquals": [
+              "States.ALL"
+            ],
+            "IntervalSeconds": 5,
+            "MaxAttempts": 3
+          }
+        ],
+        "End": true
+      }
+    }
+  }
+}
+JSON
 ```
 
 Again, please make sure to modify the value CNMResponseStream to match the stream name (not ARN) for your Kinesis response stream.
 
-### Task Configuration
+### Lambda Configuration
 
-The following tasks are required to be defined in the `lambdas.yml` configuration file.
+To execute this workflow, you're required to include several lambda resources in your deployment.  To do this, add the following task (lambda) definitions to your deployment along with the workflow you created above:
 
-If you're using a deployment based on the [example deployment](https://github.com/nasa/cumulus/tree/master/example) these lambdas should already be defined for you.
+* https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/cnm_to_cma_task.tf
+* https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/cnm_response_task.tf
 
-#### CNMToCMA
+**Please note:** To utilize these tasks you need to ensure you have a compatible CMA layer.  See the [deployment instructions](../deployment/README.md) for more details on how to deploy a CMA layer.
```

Comment:
Cant we link to the exact header here?

Reply:
Sure, when this was written I wasn't sure what that would be.

Reply:
826ca9ec3